src/colors/colors.css

```diff
@@ -1,7 +1,12 @@
 :root {
-  --omrs-primary-blue: #1a73e8;
-  --omrs-text-black: #212121;
-  --omrs-inactive-grey: #5f6369;
+  --omrs-primary-blue: #1e4bd1;
+  --omrs-text-black: #121212;
+  --omrs-inactive-grey: #3c3c43;
   --omrs-background-canvas: #fafafa;
-  --omrs-critical-alert-red: #e13332;
+  --omrs-critical-alert-red: #b50706;
+  --omrs-success-alert-green: #006800;
```

Comment:
We should add the new colors to https://github.com/openmrs/openmrs-esm-styleguide/blob/03ff09f407f8a1e0e9e607a6351ed096d97d320b/src/colors/colors.stories.js#L9

Reply:
This will be addressed by a different PR.